Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -71,13 +71,12 @@ public void ProcRunner_FailsOnTimeout()
         {
             // Arrange
 
-            // Previously the script used "TIMEOUT 2" to pause execution in
-            // the spawned process. However, the test was failing on some machines
-            // with the error "Input redirection is not supported, exiting the process immediately."
-            // Pinging a non-existent address with a timeout is an alternative 
-            // that works if the machine is connected to the internet
+            // Calling TIMEOUT can fail on some older OSes (e.g. Windows 7) with the error 
+            // "Input redirection is not supported, exiting the process immediately."
+            // However, it works reliably on the CI machines. Alternatives such as
+            // pinging a non-existent address with a timeout were not reliable.
             string exeName = TestUtils.WriteBatchFileForTest(TestContext,
-@"@ping 1.1.1.1 -n 1 -w 2000 >nul
+@"TIMEOUT 1
```

Comment:
Switching the test back to use TIMEOUT, which was reliable on the CI machines. The test passed a dozen times in succession locally.